content/all/react-testing-4-cypress.md

```diff
@@ -490,17 +563,25 @@ With the Cypress’ [viewport()](https://docs.cypress.io/api/commands/viewport.h
 });
 ```
 
-### Debugging
+## Debugging
 
-Cypress docs has a thorough [debugging guide](https://docs.cypress.io/guides/guides/debugging.html).
+Cypress docs have a thorough [debugging guide](https://docs.cypress.io/guides/guides/debugging.html).
```

Comment:
I think **has** is better here as you refer to Cypress Documentation 🤔

Reply:
Grammarly may be confused by the word “docs”, I'm not sure what's correct.